### PR TITLE
fix(curriculum): remove helpRoom and fileName

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
Neither are used any more.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: curriculum/challenges/_meta/rosetta-code/meta.json [hunk 1]
```diff
@@ -649,8 +649,4 @@
       "594810f028c0303b75339ad8",
       "Zig-zag matrix"
     ]
-  ],
-  "helpRoom": "",
-  "nChallenges": 437,
```
Comment (ShaunSHamilton) [inline curriculum/challenges/_meta/rosetta-code/meta.json:654]: Was this intentional?

Comment (ojeytonwilliams) [reply curriculum/challenges/_meta/rosetta-code/meta.json:654]: Nope. Regex mistake on my part, but I suspect that can go too.

Comment (ojeytonwilliams) [reply curriculum/challenges/_meta/rosetta-code/meta.json:654]: It's only declared here and used nowhere, so I'm happy with it going away.

**Comment by ojeytonwilliams** [comment]
> Are we going to leave `helpRoom` and `fileName` in `api-server/src/common/models/challenge.json`?

Good question.  @raisedadead any idea if we'd break something in the DB if we removed these?

**Comment by ShaunSHamilton** [comment]
Also, I do not see it being used anywhere here: https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/_meta/apis-and-microservices-certificate/meta.json#L18

**Comment by ojeytonwilliams** [comment]
Right.  None of these properties are used by the code any more.  The only concern is what this means for the DB.  Do we need to migrate or can we simply drop them?

**Comment by ShaunSHamilton** [comment]
> Do we need to migrate or can we simply drop them?

I suppose we have two things to keep in mind:
1) Dropping could create more schemas for Mrugesh to play with
2) For TS, fetching from DB could cause data to contain these _dropped_ properties. Perhaps confusing, if we forget, and do not migrate.

**Comment by raisedadead** [comment]
The `challenge` model (or its collection in MongoDB) is not in use. The whole model can go, but we should not worry about that in this PR. It would be a part of the API migration.

**Review by raisedadead — DISMISSED** [review]
Good to go if all bases have been covered in the client and the curriculum packages.

**Comment by ojeytonwilliams** [comment]
Perfect, thanks for the explanation @raisedadead

**Comment by ShaunSHamilton** [comment]
Only thing is `fileName` still exists in all of the certificate files.

> Also, I do not see it being used anywhere here: https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/_meta/apis-and-microservices-certificate/meta.json#L18

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍 
A build might tell if Gatsby complains

**Review by raisedadead — APPROVED** [review]

**Pull request merged**